StudCity.Frontend/src/i18n/index.js

```diff
@@ -0,0 +1,32 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+import translationEN from './locales/en/translation.json';
+import translationua from './locales/ua/translation.json';
```

Comment:
I think it's better to name the ukrainian translation similar to enlish one, meaning like this translationUA

Reply:
Good catch!!! This happened because I changed all mentions of the word `'ukr'` at once ^)